homeassistant/components/heos/__init__.py

```diff
@@ -56,9 +69,22 @@ async def async_setup_entry(hass: HomeAssistant, entry: HeosConfigEntry) -> bool
         hass.config_entries.async_update_entry(entry, unique_id=DOMAIN)
 
     host = entry.data[CONF_HOST]
+    credentials: Credentials | None = None
+    if CONF_USERNAME in entry.options and CONF_PASSWORD in entry.options:
```

Comment:
```suggestion
    if entry.options:
```
if one is in the other one is as well